crates/wasmtime/src/config.rs

```diff
@@ -886,6 +900,8 @@ impl Config {
 
     /// Clears native CPU flags inferred from the host.
     ///
+    /// Note: this method will change the target to that of the host.
+    ///
```

Comment:
Perhaps we could just outright remove this method now? With `Config::target` it doesn't seem too useful any more